case_utils/case_validate/__init__.py

```diff
@@ -64,13 +65,175 @@ class NonExistentCDOConceptWarning(UserWarning):
     pass
 
 
+class ValidationResult:
+    conforms: bool
+    graph: Any
+    text: str
+    undefined_concepts: Set[rdflib.URIRef]
+
+
 def concept_is_cdo_concept(n_concept: rdflib.URIRef) -> bool:
+    """
+    Determine if a concept is part of the CDO ontology.
+
+    :param n_concept: The concept to check.
+    :return: whether the concept is part of the CDO ontology.
```

Comment:
`s/ontology/ontologies/`, but otherwise thank you for this docstring!

Reply:
Fixed